doc/sphinx-guides/source/conf.py

```diff
@@ -224,9 +224,10 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-html_js_files = [
-    'js/jquery-3.4.1.min.js',
-]
+# this breaks $
```

Comment:
should we consider just removing this, if it's not needed?

Reply:
I commented out that setting and left it as pseudo-documentation in case anyone wondered why it was removed, and in case it needed to be put back in the future for whatever reason. Happy to remove it if you think I should.

Reply:
I honestly don't know what it would need to be there for - we always have version control if we need, so I'd vote just remove.